slack-service/src/main/resources/application.yml

```
@@ -37,4 +40,5 @@ springdoc:
     path: /swagger-ui.html
 slack:
   webhook:
-    url: https://hooks.slack.com/services/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
\ No newline at end of file
+    url: ${SLACK_URL}
+  
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)